course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+# EEB430H1 Theoretical Ecology [24L, 12P]
```

Comment:
Well, just because the official course title and code are set, doesn't mean we can't use a different one on the syllabus. I've seen it done many times in courses I've taken